VideoSampleCaptureRender/ViewController.swift

```diff
@@ -327,7 +341,12 @@ extension ViewController: TWCConversationDelegate {
         
         // Create a new instance of LocalMedia and use it when returning to the listening (preview) state
         self.setupLocalMedia()
-        self.updateClientStatus(.Listening, animated: true)
+        
+        if (self.client!.listening) {
+            self.updateClientStatus(.Listening, animated: true)
+        } else {
+            self.updateClientStatus(.FailedToListen, animated: true)
+        }
```

Comment:
Factor out lines 340-349 and reuse the method when an Invite fails to result in a Conversation.